h2o-py/tests/testdir_misc/pyunit_checkpointing.py

```diff
@@ -20,9 +20,32 @@ def checkpointing_test():
     checkpointed_gbm = H2OGradientBoostingEstimator(ntrees=2, checkpoint=gbm.model_id)
     checkpointed_gbm.train(x=["Origin", "Dest"], y="Distance", training_frame=airlines, validation_frame=airlines)
     assert checkpointed_gbm.checkpoint == gbm.model_id
-    
+
+
+def checkpointing_with_delete_test():
+    """
+    The reason is to be sure that whole cluster can ho down and we are able to restart from checkpoint.
+    """
+    airlines = h2o.import_file(path=pyunit_utils.locate("smalldata/testng/airlines_train.csv"))
+    gbm = H2OGradientBoostingEstimator(ntrees=1)
+    gbm.train(x=["Origin", "Dest"], y="Distance", training_frame=airlines, validation_frame=airlines)
+
+    path = pyunit_utils.locate("results")
+
+    gbm_path = h2o.save_model(gbm, path=path, force=True)
+    h2o.remove_all([airlines.frame_id])
+    gbm = h2o.load_model(gbm_path)
+
+    checkpointed_gbm = H2OGradientBoostingEstimator(ntrees=2, checkpoint=gbm)
+    checkpointed_gbm.train(x=["Origin", "Dest"], y="Distance", training_frame=airlines, validation_frame=airlines)
+    assert checkpointed_gbm.checkpoint == gbm
+
+    checkpointed_gbm = H2OGradientBoostingEstimator(ntrees=2, checkpoint=gbm.model_id)
+    checkpointed_gbm.train(x=["Origin", "Dest"], y="Distance", training_frame=airlines, validation_frame=airlines)
+    assert checkpointed_gbm.checkpoint == gbm.model_id
+
 
 if __name__ == "__main__":
-    pyunit_utils.standalone_test(checkpointing_test)
+    pyunit_utils.run_tests([checkpointing_test, checkpointing_with_delete_test])
```

Comment:
please either update the `else` condition below or just use `pyunit_utils.run_tests` (my preference) without the unnecessary `if __name__ == "__main__":` which is useful only to show a green triangle in IntelliJ (without it you can still run tests with keyboard shortcuts).

the fact is that the test runner used in Jenkins is sourcing the files, so no name by default… I can't remember if I fixed it by making it behave as if it was also named "__main__" in this case as we used to have different behaviors depending on how Py tests were executed.
Maybe the cleanest would be to delete the `else` clause everywhere and keep the `if __name__` one: if it's fixed in Jenkins runner then only that one makes sense.